src/fmllauncher/java/net/minecraftforge/fml/log4j/ForgeHighlight.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2020.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.fml.log4j;
```

Comment:
Shouldn't this be in the fmllauncher package? So that it gets built into the fmllauncher/outside of the modlauncher/classpath stuff?

Reply:
It's in the fmllauncher module and needs its own package for log4j exclusion

Reply:
Ah so it is, glossed over the root folder and looked at the package only. You should move it to /loading/log4j then.